currency-conversion/src/main/java/lk/ac/ucsc/currencyconversion/ForexServiceProxy.java

```diff
@@ -0,0 +1,14 @@
+package lk.ac.ucsc.currencyconversion;
+
+import org.springframework.cloud.netflix.ribbon.RibbonClient;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient(name = "forex-service")
+@RibbonClient(name = "forex-service")
+public interface ForexServiceProxy {
```

Comment:
@Binurim add doc comments describing what interface methods do. thats one of reason to create interfaces

Reply:
done. Check